src/main/java/com/bobocode/hoverla/bring/context/BeanDefinitionValidator.java

```diff
@@ -88,18 +89,17 @@ public void validate(List<BeanDefinition> beanDefinitions) {
         }
 
         for (BeanDefinition beanDefinition : beanDefinitions) {
-            validateBeanName(beanDefinition);
+            validateName(beanDefinition.name(), beanDefinition.type(), "Bean");
             validateDependencies(beanDefinition, beanDefinitions);
         }
     }
 
-    private void validateBeanName(BeanDefinition beanDefinition) {
-        String name = beanDefinition.name();
-        String type = beanDefinition.type().getName();
-        log.trace("Validating name for bean definition: {} - {}", name, type);
+    private void validateName(String name, Class<?> type, String checkInstance) {
+        String typeName = type.getName();
+        log.trace("Validating name for bean definition: {} - {}", name, typeName);
```

Comment:
it seems like
log.trace("Validating name for bean definition: {} - {}", name, typeName);
would be wrong if we want to say that dependency name is wrong, because we're checking dependency name rather then bean definition name

Reply:
Good catch. Changed